applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -29,10 +29,10 @@ namespace Kratos
     KRATOS_CREATE_VARIABLE( double, MEAN_RADIUS )
     KRATOS_CREATE_VARIABLE( int,    SECTION_SIDES )
     KRATOS_CREATE_VARIABLE( Matrix , GEOMETRIC_STIFFNESS )
-	KRATOS_CREATE_VARIABLE(Vector, LOCAL_AXIS_VECTOR_1)
-	KRATOS_CREATE_VARIABLE(Matrix, LOCAL_ELEMENT_ORIENTATION)
-	KRATOS_CREATE_VARIABLE(double, ORTHOTROPIC_ORIENTATION_ASSIGNMENT)
-	KRATOS_CREATE_VARIABLE(Vector, ORTHOTROPIC_FIBER_ORIENTATION_1)
+    KRATOS_CREATE_VARIABLE(Vector, LOCAL_AXIS_VECTOR_1)
```

Comment:
i think that we agreed that we will call the LOCAL_AXIS as 
LOCAL_AXIS_1,LOCAL_AXIS_2,LOCAL_AXIS_3
which should be kratos core variables

this would be consistent with the discussion in
https://github.com/KratosMultiphysics/GiDInterface/issues/103

Reply:
@RiccardoRossi I just added this in order to compile after the Western Schism of yesterday